tasks/cluster_agent.py

```diff
@@ -187,10 +187,18 @@ def hacky_dev_image_build(
     target_image="cluster-agent",
     push=False,
     signed_pull=False,
+    arch=None,
 ):
     os.environ["DELVE"] = "1"
     build(ctx)
 
+    if arch is None:
+        arch = CONTAINER_PLATFORM_MAPPING.get(platform.machine().lower())
+
+    if arch is None:
+        print("Unable to determine architecture to build, please set `arch`")
```

Comment:
```suggestion
        print("Unable to determine architecture to build, please set `arch`", file=sys.stderr)
```
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏To avoid polluting stdout